pkg/storaged/storage.scss

```diff
@@ -439,3 +439,15 @@ td.ct-text-align-right {
     // Overlay at 1/3  to mix the light and dark against the trough color
     opacity: 0.333;
 }
+
+@media ( max-width: $pf-global--breakpoint--md - 1 ) {
```

Comment:
Seems like Pf4 offers already classes for this https://github.com/patternfly/patternfly/blob/main/src/patternfly/sass-utilities/placeholders.scss#L70
pf-m-hidden-on, pf-m-visible-on.

Reply:
Can you show how to use them?

Reply:
From the examples I see, `pf-m-hidden pf-m-visible-on-md` if you want to show the icon for more than md breakpoint. (Did not try it)

Reply:
Thanks, I'll try that!

Reply:
I can't get this to work, sorry.  And since it is not documented, let's not use this.

Reply:
Ok no worries about it.